docs/resources/database.md

```diff
@@ -49,14 +49,14 @@ resource "snowflake_database" "from_share" {
 
 ### Required
 
-- `name` (String)
+- `name` (String) Specifies the identifier for the database; must be unique database name.
 
 ### Optional
 
-- `comment` (String)
-- `data_retention_time_in_days` (Number) Number of days for which Snowflake retains historical data for performing Time Travel actions (SELECT, CLONE, UNDROP) on the object. A value of 0 effectively disables Time Travel for the specified database. Default value for this field is set to -1, which is a fallback to use Snowflake default. For more information, see Understanding & Using Time Travel.
+- `comment` (String) Specifies a comment for the database.
+- `data_retention_time_in_days` (Number) Number of days for which Snowflake retains historical data for performing Time Travel actions (SELECT, CLONE, UNDROP) on the object. A value of 0 effectively disables Time Travel for the specified database. Default value for this field is set to -1, which is a fallback to use Snowflake default. For more information, see [Understanding & Using Time Travel](https://docs.snowflake.com/en/user-guide/data-time-travel).
```

Comment:
To change the documentation, you have to change the `Description` field in the database schema (you partially modified it), then you run `make docs` to re-generate documentation which is generated from all of the schemas. Any file under `docs/` is generated, so any change you made there and didn't come from schema Description change will be overriden by us in the next pr-s.